src/api/TokenIO.h

```diff
@@ -50,6 +50,7 @@
  * @param developerKey developer ID
  * @param cryptoEngineFactory crypto module to use
  * @param useSsl use SSL if true
+ * @param certsPath use custom certs if set
```

Comment:
...otherwise, use default root certs (is it root certs, not some other kind? I just copy-pasted from slack, I dont know these things) (uhm, where I can I see these? is https://github.com/grpc/grpc/blob/master/etc/roots.pem what I want? so if I want to use this option, I should copy that file and paste mine at the end?)

Reply:
nice catch.